src/bindings/c/include/openvino/c/ov_common.h

```diff
@@ -33,7 +33,7 @@
 #ifdef __cplusplus
 #    define OPENVINO_C_API_EXTERN extern "C"
 #else
-#    define OPENVINO_C_API_EXTERN
+#    define OPENVINO_C_API_EXTERN extern
```

Comment:
I think it's required only for `OPENVINO_C_VAR` ? while functions are declared in "extern" fashion by definition.